pkgs/by-name/dw/dwlb/package.nix

```
@@ -0,0 +1,48 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  wayland-scanner,
+  wayland-protocols,
+  pixman,
+  fcft,
+  wayland,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dwlb";
+  version = "unstable-2024-09-11";
+
+  src = fetchFromGitHub {
+    owner = "kolunmi";
+    repo = "dwlb";
+    rev = "0daa1c1fdd82c4d790e477bf171e23ca2fdfa0cb";
+    hash = "sha256-Bu20IqRwBP1WRBgbcEQU4Q2BZ2FBnVaySOTsCn0iSSE=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  env = {
+    PREFIX = "$(out)";
```

```suggestion
    PREFIX = placeholder "out";
```

not necessary in this case ,but for education purpose